gunicorn/config.py

```diff
@@ -465,11 +465,13 @@ class ConfigFile(Setting):
     name = "config"
     section = "Config File"
     cli = ["-c", "--config"]
-    meta = "FILE"
+    meta = "CONFIG"
     validator = validate_string
     default = None
     desc = """\
-        The path to a Gunicorn config file, or python module.
+        The Gunicorn config file.
+
+        A string of the form 'PATH', 'file:PATH', or 'python:MODULE_NAME'.
```

Comment:
Okay. Note that other descriptions, such as [bind](https://github.com/benoitc/gunicorn/blob/72a8e2940d75b3ef7a92edcb3e4247f43426706f/gunicorn/config.py#L494) and [syslog_addr](https://github.com/benoitc/gunicorn/blob/72a8e2940d75b3ef7a92edcb3e4247f43426706f/gunicorn/config.py#L1157), use single quotes as well.